0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    new_key, subkey = random.split(key)
</pre><p>I haven't used JAX before, so let. me know if this is a naming convention. It might be better to give slightly more intuitive names for the keys. Something like "new_split_key" (one we will continue to split) and "new_generator_key" (one we will use to generate a new random number).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Yeah, I think this more the naming covention thing. Usually, people just say <code>key, subkey = ...</code>.  I added <code>new_key</code> to be more explicit.   </p>